src/main/java/io/pakland/mdas/githubstats/domain/model/HistoricQueries.java

```diff
@@ -1,25 +1,42 @@
 package io.pakland.mdas.githubstats.domain.model;
 
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
 import javax.persistence.*;
 
+@Data
+@NoArgsConstructor
 @Entity
-@Table(name = "historic_queries")
+@Table(name = "\"historic_queries\"")
```

Comment:
As a question, what's the reason to escape quotes in this cases?

Reply:
Any idea @sdomingobasora? I would like to know this so that we can close the PR.

Reply:
@mikededo you are right, these quotes can be deleted, don't add any value

Reply:
No idea @mikededo. I would trust that it's there for a solid reason (otherwise it looks odd to me). Only reason I can vaguely correlate with this is this piece of the issue description

> Minor problems were found because some fields named reserved words in SQL like 'from' or 'to' and were needed to escape them.

I would approve the changes for now because some changes in this PR are relevant for other use cases and thus it'd be useful to have them committed. We can discuss these escape quotes at class if they pose some issue